dashboards/heroku/postgres.json

```diff
@@ -367,6 +365,64 @@
             ]
           }
         ]
+      },
+      {
+        "type": "timeseries",
+        "display": "line",
+        "title": "WAL usage",
+        "description": "Write Ahead Log usage, in percentage. A healthy database has this metric at 0.75 or lower. Higher than 0.75 Heroku automatically limits connections to the Heroku Postgres instance.",
+        "line_label": "%source%",
```

Comment:
What are the chances of there being multiple addons and two lines with the same source tag being shown? Shouldthe addon tag also be part of the label?

Same for the other graph.

Reply:
Those would still have a different `source` tag, I believe (e..g the source is always different, the addon tag can be the same).

We don't use the addon tag in any of the other graphs, so I kept it the same. We haven't had any complaints about it so I'd assumed it's fine.